scripts/kind-e2e/e2e.sh

```diff
@@ -1,76 +1,26 @@
+## Process command line flags ##
 
-### Functions ###
+source /usr/share/shflags/shflags
+DEFINE_string 'globalnet' 'false' "Deploy with operlapping CIDRs (set to 'true' to enable)"
+DEFINE_string 'lighthouse' 'false' "Deploy with lighthouse"
+DEFINE_string 'status' 'onetime' "Status flag (onetime, create, keep, clean)"
+FLAGS "$@" || exit $?
+eval set -- "${FLAGS_ARGV}"
 
-function kind_clusters() {
-    status=$1
-    lighthouse=$2
-    globalnet=$3
-    version=$4
-    pids=(-1 -1 -1)
-    logs=()
-    for i in 1 2 3; do
-        if [[ $(kind get clusters | grep cluster${i} | wc -l) -gt 0  ]]; then
-            echo Cluster cluster${i} already exists, skipping cluster creation...
-        else
-            logs[$i]=$(mktemp)
-            echo Creating cluster${i}, logging to ${logs[$i]}...
-            (
-            if [[ -n ${version} ]]; then
-                kind create cluster --image=kindest/node:v${version} --name=cluster${i} --config=${PRJ_ROOT}/scripts/kind-e2e/cluster${i}-config.yaml
-            else
-                kind create cluster --name=cluster${i} --config=${PRJ_ROOT}/scripts/kind-e2e/cluster${i}-config.yaml
-            fi
-            master_ip=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster${i}-control-plane | head -n 1)
-            sed -i -- "s/user: kubernetes-admin/user: cluster$i/g" $(kind get kubeconfig-path --name="cluster$i")
-            sed -i -- "s/name: kubernetes-admin.*/name: cluster$i/g" $(kind get kubeconfig-path --name="cluster$i")
-            sed -i -- "s/current-context: kubernetes-admin.*/current-context: cluster$i/g" $(kind get kubeconfig-path --name="cluster$i")
-
-            if [[ ${status} = keep ]]; then
-                cp -r $(kind get kubeconfig-path --name="cluster$i") ${PRJ_ROOT}/output/kind-config/local-dev/kind-config-cluster${i}
-            fi
-
-            sed -i -- "s/server: .*/server: https:\/\/$master_ip:6443/g" $(kind get kubeconfig-path --name="cluster$i")
-            cp -r $(kind get kubeconfig-path --name="cluster$i") ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster${i}
-            ) > ${logs[$i]} 2>&1 &
-            set pids[$i] = $!
-        fi
-    done
-    if [[ ${#logs[@]} -gt 0 ]]; then
-        echo "(Watch the installation processes with \"tail -f ${logs[*]}\".)"
-        for i in 1 2 3; do
-            if [[ pids[$i] -gt -1 ]]; then
-                wait ${pids[$i]}
-                if [[ $? -ne 0 && $? -ne 127 ]]; then
-                    echo Cluster $i creation failed:
-                    cat ${logs[$i]}
-                fi
-                rm -f ${logs[$i]}
-            fi
-        done
-    fi
-    KUBECONFIG=${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster1:${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster2:${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster3 \
-    kubectl config view --flatten > ${PRJ_ROOT}/output/kind-config/dapper/kind-config-merged
```

Comment:
I suspect we still need this somewhere...

Reply:
I kept it, it's in the main section

Reply:
Indeed, I’m blind and I don’t know how to use Ctrl+F!